UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Common/Helpers/NMHelper.cs

```diff
@@ -125,5 +132,24 @@ public static async Task<byte[]> GetFileBytesFromStream(Stream stream)
             stream.Close();
             return fileBytes;
         }
+
+        public static string GetYearAndTypeFromFilenName(string fileName)
+        {
```

Comment:
I think, there is no need of this method to order by year and file name for cumulative files, Year & DataDate attributes can be used for same.

Reply:
Yes removed this function and related unit test case \s